azure-pipelines-expectation-gallery.yml

```diff
@@ -0,0 +1,168 @@
+# This file is responsible for configuring the `expectation_gallery` pipeline (https://dev.azure.com/great-expectations/great_expectations/_build)
+#
+# The pipeline is run under the following conditions:
+#   - On the develop branch when a weekly release is being cut
+#   - On the develop branch as scheduled by the below cron job
+#
+# The sole purpose of this pipeline is to build and publish the Expectation gallery. As such, it is designed to run quickly and frequently
+# to ensure that the gallery is kept to-up-date with change.
```

Comment:
Runs based on the following conditions:
* On schedule (every 2 hours)
* On release cut
* If run manually

Note that if no files have changed, it will skip the run.